test/groovy/ChecksPublishResultsTest.groovy

```diff
@@ -0,0 +1,181 @@
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.RuleChain
+
+import com.lesfurets.jenkins.unit.BasePipelineTest
+
+import static org.junit.Assert.assertEquals
+import static org.junit.Assert.assertTrue
+
+import util.Rules
+
+class ChecksPublishResultsTest extends BasePipelineTest {
+    Map publisherStepOptions
+    List archiveStepPatterns
+
+    @Rule
+    public RuleChain ruleChain = RuleChain.outerRule(Rules.getCommonRules(this))
```

Comment:
What is the reason for wrapping the RuleChain returned by ```Rules.getCommonRules(./.)``` inside another rule chain?

Reply:
there is none 👍